docs/development_controllers/chip-tool/chip_tool_guide.md

```diff
@@ -9,33 +9,27 @@ the setup payload or performing discovery actions.
 
 <hr>
 
-## Source files
+## Installation
 
-You can find source files of the CHIP Tool in the `examples/chip-tool`
-directory.
-
-> **Note:** The CHIP Tool caches the configuration state in the
-> `/tmp/chip_tool_config.ini` file. Deleting this and other `.ini` files in the
-> `/tmp` directory can sometimes resolve issues related to stale configuration.
+On devices running SnapD, like Ubuntu, the CHIP Tool can be installed from the
+Snap Store. To do this, run:
 
-> **Note:** To make the configuration persistent (since `/tmp` directory might
-> be flushed at each reboot) you can change the directory where CHIP Tool caches
-> its configuration by using the option `--storage-directory`
+```
+sudo snap install chip-tool
```

Comment:
who maintains this snap build? How can people ensure its authenticity and safety?

Reply:
Link to the respository for issue reporting and tracking may also be useful

Reply:
We've added a link to the [snap store listing](https://snapcraft.io/chip-tool). This listing indicates that the maintainer of the snap is Canonical - a [CSA member](https://csa-iot.org/member/canonical/). The listing also links to the [source code](https://github.com/canonical/chip-tool-snap) of the snap package on Github.